sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.8.0</version>
```

Comment:
Two questions still -- should this not just use the same version of Avro as the rest of the project? `avro.version` is already defined to be 1.7.7 in the parent POM. I'd think we need to use that instead.

Second, is this really just a test issue? `sql/hive` depends on `avro`. It may be that really `sql/core` needs to depend on `avro` too -- in non-test scope. In fact, `mvn dependency:tree` shows that `sql/core` already depends on it in `compile` scope transitively.

Therefore, I wonder if it is best to simply remove the `version` and `scope` here. That may be the most correct thing?